content/projects/data-science-specific/data-visualisation/mobile-money-viz/_index.md

```diff
@@ -38,15 +38,17 @@ This dataset is the geospatial mapping of all cash outlets in Tanzania in 2012.
 
 ## Instructions:
 
-1. Examine the dataset. Are there any missing observations or columns where the data do not seem valid?
+1. Import the required libraries and load the datadatset [training](training.csv).
+   
+2. Explore the dataset. Are there any missing observations or columns where the data do not seem valid?
 
-2. Get basic descriptive statistics for the dataset.
+3. Get basic descriptive statistics for the dataset.
 
-3. Create appropriate graphs to visually represent the relationship between financial services accessed (non-mobile, mobile, both) and age, gender, marital status, land ownership and type of income.
+4. Create appropriate graphs to visually represent the relationship between financial services accessed (non-mobile, mobile, both) and age, gender, marital status, land ownership and type of income. Provide interpretation for each graph.  
```

Comment:
"Provide an interpretation for each graph."  It's important to use valid English sentences

Reply:
Sure, done.